test/test_jld2_output_writer.jl

```diff
@@ -150,8 +152,19 @@ for arch in archs
 
         simulation = Simulation(model, Δt=1.0, stop_iteration=1)
 
-        simulation.output_writers[:velocities] = JLD2OutputWriter(model, model.velocities,
-                                                                  schedule = TimeInterval(1),
+        # Flavors of output: functions, AbstractOperations, FunctionFields
+        clock = model.clock
+        α = 0.12
+        test_function_field = FunctionField{Center, Center, Center}((x, y, z, t, α) -> α * t; clock, parameters=α)
```

Comment:
```suggestion
        test_function_field = FunctionField{Center, Center, Center}((x, y, z, t, α) -> α * t, grid; clock, parameters=α)
```

Reply:
The change above should fix tests. @glwagner do you mind if I commit this change and merge if tests pass?